src/components/projects.ts

```diff
@@ -28,6 +28,16 @@ export default [
     github: "https://github.com/WeatherXM",
     twitter: "https://twitter.com/WeatherXM",
     details: `WeatherXM is a network that's powered by community owned devices, purpose built by WeatherXM for weather data collection. As part of the network, devices are incentivized to share data, which is sent to various stations throughout the globe.\n\nOne of the challenges with this approach is not only ensuring that the raw device data is available when it's needed, but as a decentralized network, its infrastructure must also make sure that data is stored in a decentralized fashion instead of centralized infrastructure.\n\nAs devices send data to the network, it's replicated to a hot for immediate and TTL retrieval, which is pertinent for compute over that data. And for persistence guarantees, this data is also replicated to Filecoin for long-term storage guarantees, which can help the network ensure that data is available for years to come.`,
+    blogs: [
```

Comment:
small optional feature to show links to blog posts about partners, if available (only wxm has anything)